### ISSUE TITLE
Max CPU freq you are using in this project.

### ISSUE THREAD

**Issue by shirish47** [PetteriAimonen/ED060SC4_driver#2]
We have developed some boards and are trying to drive the screen with this drivers.. when trying to do sub clear we used to get barcode kind of view on screen. anyways I want to know what is the  Freq of your CPU? 
spritesmode pushes EPS8266 at 160Mhz at some point. May be because of the Timing and shift Registers.

Teensy3.1 can max at 92Mhz and I think the note you wrote says We have to get the delay in atkeast to 50nS.

**Comment by shirish47** [comment]
when I did digitalWrite(x,HIGH) and then no delay DigitalWrite(LOW) in loop I was able to get 240ns high time.. but then I set GPIOD_PDOR=0x1 //pin 2 on Teensy3.1 and then GPIOD_PDOR=0x0 I got to 41.25ns of high time. I hope this can help us further.

**Comment by PetteriAimonen** [comment]
I'm running STM32L151 at 16 MHz. The timings are not really that 
critical, faster cpu will mean faster screen update.

20 MHz (50 ns period) is the maximum pixel clock frequency, i.e. that is 
the fastest speed at which you can update. If you are not in a hurry, 
you can easily go down to 1MHz or even less without any trouble.

**Comment by shirish47** [comment]
oh ok... I thought you wrote "50 ns atleast"

**Comment by PetteriAimonen** [comment]
Yeah, "50 ns atleast" means 50 ns or longer time, so 20 MHz or lower 
frequency :)

**Comment by shirish47** [comment]
you can check things here : https://github.com/shirish47/ED060SC4

I have uploaded code I am trying and some files logic files. Please check the logic file in saleae logic software and tell me are the signal sequences correct? also check the .ino code.. I am doing pixel write and then flush ... also data lines change but nothing happens on screen accept some barcode like things happen on screen.

**Comment by PetteriAimonen** [comment]
Sorry, but I don't currently have time to help that much.

Getting atleast something happening is good. Depending on whether the 
lines are horizontal or vertical, you can figure out whether the problem 
is in the vertical or horizontal drivers, and then try to debug that.
